src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -219,7 +215,7 @@ int main(int argc, char **argv)
     Image<unsigned char> imageMask;
 
     const std::string sGlobalMask_filename = stlplus::create_filespec(sfm_data.s_root_path, "mask.png");
```

Comment:
Here you are using the main image path => -i
But in the doc you was mentioning the path of the sfm_data file => -o
Which one do you think is the better?